caesars-palace/blocks/slider/slider.js

```diff
@@ -0,0 +1,323 @@
+      }
+    });
+    block.style = `width: calc((100%/${numberOfCardsDisplayed} - 10px)); gap: 10px;`;
+    div.classList.add('card');
+
+    const picture = div.querySelector('picture');
+    if (picture) {
+      picture.classList.add('card-image');
+      const imageParent = picture.closest('div');
+      imageParent.classList.add('card-image-parent');
+    }
+
+    const contentDivs = div.querySelectorAll(':scope > div:not(.card-image-parent)');
+    contentDivs[0].classList.add('short-description');
+    contentDivs[0].classList.add('active');
+    contentDivs[1].classList.add('long-description');
+
+    const closeButton = document.createElement('div');
+    closeButton.classList.add('close-button');
+    closeButton.classList.add('hide');
+    div.insertBefore(closeButton, contentDivs[1]);
+  });
+
+  const shortDescriptionDivs = block.querySelectorAll('.short-description');
+  shortDescriptionDivs.forEach((div) => {
+    const title = div.querySelector('h4');
+    title?.classList.add('title');
+    const subtitle = div.querySelector('h5');
+    subtitle?.classList.add('subtitle');
+    const showMore = div.querySelector('p > strong');
+    if (showMore) {
+      showMore.classList.add('show-more');
+    }
+  });
+
+  const mobileMediaQuery = window.matchMedia('only screen and (max-width:480px)');
+  const mobileMediaWidthChangeHandler = (event) => {
+    if (event.matches) {
+      const shortDesriptionTabletDivs = block.querySelectorAll('.short-description');
+      shortDesriptionTabletDivs.forEach((div) => {
+        if (!div.classList.contains('active')) {
+          div.classList.toggle('active');
+        }
+      });
+
+      const longDescriptionDivs = block.querySelectorAll('.long-description');
+      longDescriptionDivs.forEach((div) => {
+        if (div.classList.contains('active')) {
+          div.classList.toggle('active');
+        }
+      });
+      const sliderCard = block.closest('.slider');
+      sliderCard.style = 'width: 100%;';
+    }
+  };
+
+  mobileMediaWidthChangeHandler(mobileMediaQuery);
+  mobileMediaQuery.addEventListener('change', mobileMediaWidthChangeHandler);
+
+  const mediaQuery = window.matchMedia(
+    'only screen and (min-width: 481px) and (max-width:768px)',
+  );
+
+  const mediaWidthChangeHandler = (event) => {
+    if (event.matches) {
+      const shortDesriptionTabletDivs = block.querySelectorAll('.short-description');
+      shortDesriptionTabletDivs.forEach((div) => {
+        if (div.classList.contains('active')) {
+          div.classList.toggle('active');
+        }
+      });
+
+      const longDescriptionDivs = block.querySelectorAll('.long-description');
+      longDescriptionDivs.forEach((div) => {
+        if (!div.classList.contains('active')) {
+          div.classList.toggle('active');
+        }
+      });
+
+      const sliderCard = block.closest('.slider');
+      sliderCard.style = 'width: 100%;';
+    }
+  };
+  let isDragging = false;
```

Comment:
add a blank line above this line maybe?

Reply:
Actually could all these vars be moved down further so they are closer to the functions that use them. I think all these vars are for tracking dragging?